salt/states/win_appx.py

```diff
@@ -0,0 +1,125 @@
+"""
+Manage Microsoft Store apps on Windows. Removing an app with this modules will
```

Comment:
same here versionadded for this new state module

Reply:
Done